src/app/app.routing.ts

```diff
@@ -8,6 +8,7 @@ import { CommonLayoutComponent } from './layouts/common-layout/common-layout.com
 import { PasswordResetComponent } from './pages/password-reset/password-reset.component';
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { NoAuthGuard } from './guards/no-auth.guard';
+import { PaymentsComponent } from "./pages/payment/internationalPayment.component";
```

Comment:
@patrickf949 As agreed during my sync with you, let's have a `payment.routing.ts` and a `payment.module.ts` files in the **payment** folder and register all payment-related routes and components there.